run_tests.py

```diff
@@ -0,0 +1,11 @@
+#!/usr/bin/env python3
+
+import subprocess
+import os
+from pathlib import Path
+import sys
+
+env = os.environ.copy()
+env["PYTHONPATH"] = str(Path(__file__).parent)
+result = subprocess.run([sys.executable, "./tests/tests_runner.py"], env=env)
```

Comment:
It would be better not using relative path here, or at least relative to 'run_tests.py' always.
Running as `depthai-host/run_tests.py` for example will fail.
```
file_path = Path(__file__).parent / "./tests/tests_runner.py"
result = subprocess.run([sys.executable, file_path], env=env)
```
should do it.

Would be nice to make our scripts callable from any folder.

Reply:
Agreed, done (also changed filepath in requirements check)